internal/app/pacts/myconsumer-myprovider.json

```diff
@@ -6,34 +6,6 @@
     "name": "MyProvider"
   },
   "interactions": [
-    {
```

Comment:
Why do we want to delete this interaction?

Reply:
@sofib-form3 I think you added this commit, can you remember why this was necessary?

Reply:
yes, commit came from me - I cannot remember however why interaction was deleted, we should recheck it

Reply:
I've put it back on my fork - resync?

Reply:
This file is updated whenever the test are run, we have marked it as git ignored